packages/db/src/search/BaseSearch.ts

```diff
@@ -458,7 +458,13 @@ abstract class BaseSearch {
     const { sort } = dbQueryParameters || this.dbQueryParameters;
     sort?.forEach((key) => {
       if (key.column.startsWith('error')) {
-        searchQuery.orderByRaw(`${this.tableName}.error ->> 'Error' ${key.order}`);
+        searchQuery.orderByRaw(
+          `${this.tableName}.error ->> 'Error' ${key.order}`
+        );
+      } else if (dbQueryParameters?.collate) {
+        searchQuery.orderByRaw(
+          `${key} collate \"${dbQueryParameters.collate}\"`
```

Comment:
Is there enough justification to use `collate` on the base model as opposed to overwriting the child class' buildSortQuery for just e.g. recon reports?

Reply:
I think so, yes - this is generally useful in the case that sorting order matters with respect to CMR or other external data sources.